index.ts

```diff
@@ -29,11 +33,17 @@ declare module 'vue' {
 interface VueI18NextOptions {
     i18next: i18n;
     rerenderOn?: ('languageChanged' | 'loaded' | 'added' | 'removed')[];
+    // Optional custom pattern for matching slot start of the `TranslationComponent.
```

Comment:
I think you also wanted to have a closing back-tick after `TranslationComponent`.